spec/factories/organizations.rb

```diff
@@ -7,5 +7,11 @@
         create(:device_group, organization: organization)
       end
     end
+
+    factory :organization_with_device do
```

Comment:
what's a name is this?
`create(:organization, :organization_with_device)` ? looks and sounds weird

Reply:
organization may have many devices..